specify_manual/command_design/show_annotattion/useage.md

```
@@ -3,32 +3,31 @@
 In this section, we describe about how to use the show-command.
 
 /// [@st-manual-usage-show-all] layer: spec-detail, type: Convention, name: Show All Annotations
-To show all annotations (scan both `src/` and `specify_manual/`):
+To show all annotations, choose the config that matches the application and scan both document and code roots:
 ```bash
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)